src/node/coinstats.cpp

```diff
@@ -24,31 +24,35 @@ static uint64_t GetBogoSize(const CScript& scriptPubKey)
            scriptPubKey.size() /* scriptPubKey */;
 }
 
-static void ApplyStats(CCoinsStats& stats, CHashWriter& ss, const uint256& hash, const std::map<uint32_t, Coin>& outputs)
+static void ApplyHash(CCoinsStats &stats, CHashWriter& ss, const uint256& hash, const std::map<uint32_t, Coin>& outputs, std::map<uint32_t, Coin>::const_iterator it)
```

Comment:
In commit "refactor: Seperate hash and stats calculation in coinstats"

Nit: seperate -> separate in commit title

Reply:
Fixed